python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -6200,6 +6200,7 @@ def pytest_unconfigure(config):
     "tests/indexes/datetimes/test_constructors.py::TestDatetimeIndex::test_constructor_with_int_tz[US/Pacific-datetime64[ns, US/Pacific]-array-Index]",
     "tests/indexes/datetimes/test_constructors.py::TestDatetimeIndex::test_constructor_with_int_tz[US/Pacific-datetime64[ns, US/Pacific]-list-DatetimeIndex]",
     "tests/indexes/datetimes/test_constructors.py::TestDatetimeIndex::test_dti_constructor_with_dtype_object_int_matches_int_dtype[M8[us, US/Pacific]]",
+    "tests/indexes/datetimes/test_constructors.py::TestDatetimeIndex::test_dti_constructor_with_non_nano_now_today",
```

Comment:
I think this one is just a flaky test, so we should probably add this in `NODEIDS_THAT_FLAKY_XFAIL_WITH_CUDF_PANDAS`. In pandas 3.0, part of this test is marked as xfail since it compares "now" to "today" with a tolerance https://github.com/pandas-dev/pandas/blob/98c9c7f838d223e66f1459fab23ac178b29c2227/pandas/tests/indexes/datetimes/test_constructors.py#L1033

Reply:
Good call, changed it.